lib/diagnostic.rb

```diff
@@ -78,10 +93,10 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = "I am a HouseCat, and I go meow"
 
 ## Explain why this would be the output, based on the method lookup chain.
-# your response as a comment here
+# HouseCat inherits from cat that inherits from Animal - #self calls the Instatiate method of HouseCat and @sound calls the sound of the HouseCat
```

Comment:
`self` refers to the caller and doesn't call the constructor.  `@sound` is an instance variable on the constructors of `Animal` and `HouseCat`.